gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpRequestRunnableTest.java

```diff
@@ -138,6 +139,7 @@ public String serialize(Void response) {
             .setApiMethodDescriptor(methodDescriptor)
             .setHttpTransport(new MockHttpTransport())
             .setJsonFactory(new JacksonFactory())
+            .setResponseFuture(SettableApiFuture.<Void>create())
```

Comment:
This is weird. Is it a bug fix or a workaround for `responseFuture` becoming non-nullable because of `@AutoValue`? Adding `@Nulable` annotation to `abstract SettableApiFuture<ResponseT> getResponseFuture();` can fix that. Please make the change and remove this line, if the response future is actually nullable.

Reply:
The response future should not be nullable. 

It seems common in google-cloud-java to have a Void type that represents the response type.